### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by lisamartelly**

### CODE REVIEW

File: src/ProgressCircle.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import "./ProgressBar.scss";
+
+function ProgressCircle(props) {
```
Comment (danmartelly) [inline src/ProgressCircle.js:3]: Same deal as bus stop: I think it's more readable/usable when you unpack the props

File: src/Transit.js [hunk 2]
```diff
@@ -18,9 +18,17 @@ function Transit() {
   
       for (let i = 0; i < json.departures.length; i++) {
         if (json.departures[i].actual) {
-          realTimeDepartures.push(json.departures[i])
+          if (realTimeDepartures.length < 6) {
```
Comment (danmartelly) [inline src/Transit.js:21]: I think it could be a little more readable to put this condition as part of the loop check

File: src/Transit.js [hunk 3]
```diff
@@ -18,9 +18,17 @@ function Transit() {
   
       for (let i = 0; i < json.departures.length; i++) {
```
Comment (danmartelly) [inline src/Transit.js:19]: I might use some of the array functions like filter:
```
realTimeDepartures = json.departures.filter(d => d.actual).slice(0,6);
scheduledDepartures = json.departures.filter(d => !d.actual).slice(0,6);
```

File: src/stylesheets/BusStop.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.BusStop {
```
Comment (danmartelly) [inline src/stylesheets/BusStop.css:1]: I think you committed compiled code??? This should not be part of the repo. You can modify the .gitignore to make a rule that ignores this folder

File: src/stylesheets/BusStop.css.map [hunk 5]
```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["../BusStop.scss","../_variables.scss"],"names":[],"mappings":"AAEA;EACI;EACA;EACA;;;AAGJ;EACI;;;AAGJ;EACI,kBCXS;EDYT","file":"BusStop.css"}
```
Comment (danmartelly) [inline src/stylesheets/BusStop.css.map:1]: ditto

**Review by danmartelly — APPROVED** [review]
a couple of comments, but looks good

File: src/stylesheets/BusStop.css (hunk 4 above)
Comment (lisamartelly) [reply src/stylesheets/BusStop.css:1]: gotcha!

**Pull request merged**